tests/safeds_stubgen/stubs_generator/__snapshots__/test_generate_stubs.ambr

```diff
@@ -0,0 +1,152 @@
+# serializer version: 1
+# name: test_class_attribute_creation
+  '''
+  package test_stub_generation.attribute_module
+  
+  class A {}
+  
+  class B {
+  	attr type_hint_public: Int
```

Comment:
Our name convention is to use `lowerCamelCase` for the names of attributes. To still properly map the attribute to Python, add a call of the `PythonName` annotation:

```suggestion
    @PythonName("type_hint_public")
  	attr typeHintPublic: Int
```

The validation from the VS Code extension will tell you the name convention for other types of declarations. The fix is always the same.